README.md

```diff
@@ -37,42 +37,81 @@ DeepRank2 extensive documentation can be found [here](https://deeprank2.rtfd.io/
   - [Overview](#overview)
   - [Table of contents](#table-of-contents)
   - [Installation](#installation)
-    - [Dependencies](#dependencies)
-    - [Deeprank2 Package](#deeprank2-package)
-    - [Test installation](#test-installation)
+    - [Dockerfile](#dockerfile)
+    - [Non-pythonic dependencies](#non-pythonic-dependencies)
+    - [Pythonic dependencies](#pythonic-dependencies)
+      - [Test installation](#test-installation)
     - [Contributing](#contributing)
-    - [Data generation](#data-generation)
-    - [Datasets](#datasets)
-      - [GraphDataset](#graphdataset)
-      - [GridDataset](#griddataset)
-    - [Training](#training)
+  - [Data generation](#data-generation)
+  - [Datasets](#datasets)
+    - [GraphDataset](#graphdataset)
+    - [GridDataset](#griddataset)
+  - [Training](#training)
   - [Computational performances](#computational-performances)
   - [Package development](#package-development)
 
 ## Installation
 
-The package officially supports ubuntu-latest OS only, whose functioning is widely tested through the continuous integration workflows. 
+Note that the package officially supports ubuntu-latest OS only, whose functioning is widely tested through the continuous integration workflows. 
 
-### Dependencies
+### Dockerfile
 
-Before installing deeprank2 you need to install some dependencies. We advise to use a [conda environment](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) with Python >= 3.10 installed. The following dependency installation instructions are updated as of 14/09/2023, but in case of issues during installation always refer to the official documentation which is linked below:
+In order to try out the package without worrying about your OS and without the need of installing all the required dependencies, we created a `Dockerfile` that can be used for taking care of everything in a suitable container. After having cloned the repository and installed [Docker](https://docs.docker.com/engine/install/), run the following commands from the root of the repository.
+
+Build the Docker image:
+```bash
+docker build -t deeprank2 .
+```
+
+SSH to a running container:
+
+```bash
+docker run -it --expose 3000  -p 3000:3000 deeprank2
+```
+
+Run the tutorials' notebooks from within the running container:
+```bash
+cd tutorials
+jupyter notebook --ip 0.0.0.0 --no-browser --allow-root --port 3000
+```
```

Comment:
I needed to run `sudo` before any of that. I guess it depends on permissions set up (@dsmits sent me a link to make docker not depend on sudo anymore), but maybe better to just include it in the command blocks, as I don't think it hurts.

I can't add the suggestion, as the "suggestion block" gets killed by the triple-` from the bash code block :)

Reply:
On the other hand, many users would use a super computer like snellius and wouldn't have sudo permissions. I think that this is beyond the scope of these instructions anyway.

Reply:
Good point!
Maybe at least mention that a user might need sudo permission to run. I think for many inexperienced users (which would usually want to just run the docker), it would be unclear why they get a permission error when trying to build the container if just running those lines.